src/main/java/org/fundacion/jala/converter/models/UserSQL.java

```
@@ -87,6 +91,52 @@ public static User findUserById(final int userId) {
         return userResult;
     }
 
+    /**
+     * Verifies if a username exists in the database
+     * @param username a String with the username to check
+     * @return a boolean with the response
+     */
+    public static boolean usernameExists(final String username) {
+        List<User> list = listUser();
+        Boolean usernameExists = false;
+        for (User user : list) {
+            if (username.equals(user.getName())) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Edits the token value in the database
+     * @param username a String with the username
+     * @param token a String with the token
+     */
+    public static void editToken(final String username, final String token) {
+        int userId = getUserId(username);
+        EntityManager manager = emf.createEntityManager();
+        manager.getTransaction().begin();
+        User editUser = manager.find(User.class, userId);
+        editUser.setToken(token);
+        manager.getTransaction().commit();
+        manager.close();
+    }
+
+    /**
+     * Gets the userId from the database
+     * @param username a String to look for the userId
+     * @return an int with the userId
+     */
+    public static int getUserId(final String username) {
+        List<User> list = listUser();
+        for (User user : list) {
+            if (username.equals(user.getName())) {
+                return user.getId();
+            }
+        }
+        return 0;
```

Would be better to throw an exception rather than returning a 0.